src/inspector_agent.cc

```diff
@@ -658,9 +686,21 @@ bool Agent::StartIoThread(bool wait_for_connect) {
   }
 
   v8::Isolate* isolate = parent_env_->isolate();
+  HandleScope handle_scope(isolate);
+
+  // Enable tracking of async stack traces
+  if (!enable_async_hook_function_.IsEmpty()) {
+    Local<Function> enable_fn = enable_async_hook_function_.Get(isolate);
```

Comment:
Actually I think it should be all good now, as that function calls `Connect()` which hooks up the hooks if it hasn't already.